lib/ProductOpener/Attributes.pm

```diff
@@ -299,13 +301,15 @@ sub initialize_attribute ($attribute_id, $target_lc) {
 		my $analysis_tag = $attribute_id;
 		$analysis_tag =~ s/_/-/g;
 		$attribute_ref->{icon_url} = "$static_subdomain/images/attributes/$analysis_tag.svg";
+		$attribute_ref->{panel_id} = "ingredients_analysis_en-" . $analysis_tag;
+		# does not seem to be working
 	}
 	elsif ($attribute_id =~ /^(labels)_(.*)$/) {
 		my $tagtype = $1;
 		my $tag = $2;
 		$tag =~ s/_/-/g;
-
 		$attribute_ref->{icon_url} = "$static_subdomain/images/attributes/${tag}.svg";
+		$attribute_ref->{panel_id} = "${tag}";
```

Comment:
There is "environment_label_en:" in front. And we should do this only for tags with a panel (how do we get them ? Is it through a property in taxonomy ?)

Reply:
@alexgarel would removing this change for the labels make the PR acceptable to you ? we indeed don't have any knowledge panels on labels, but that would be another issue that we can create